layer/rpi/services/rpi-connect.yaml

```diff
@@ -0,0 +1,54 @@
+# METABEGIN
+# X-Env-Layer-Name: rpi-connect
+# X-Env-Layer-Category: service
+# X-Env-Layer-Desc: Raspberry Pi Connect client with screen-sharing support and
+#  remote shell access
+# X-Env-Layer-Version: 1.0.0
+# X-Env-Layer-Requires: systemd-min,rpi-user-credentials
+# X-Env-Layer-Provides: rpi-connect-client
+#
+# X-Env-VarRequires: IGconf_device_user1
+# X-Env-VarRequires-Valid: string
+#
+# X-Env-VarPrefix: connect
+#
+# X-Env-Var-authkey:
+# X-Env-Var-authkey-Desc: Path to authentication keyfile
+# X-Env-Var-authkey-Required: n
+# X-Env-Var-authkey-Valid: string
+# X-Env-Var-authkey-Set: n
+#
+# X-Env-Var-on: y
+# X-Env-Var-on-Desc: Enable Raspberry Pi Connect on system startup
+# X-Env-Var-on-Required: n
+# X-Env-Var-on-Valid: bool
+# X-Env-Var-on-Set: y
+# METAEND
+---
+mmdebstrap:
+  packages:
+    - rpi-connect
+    - wayvnc
+  setup-hooks:
+    - |-
+      if [ -n "${IGconf_connect_authkey+x}" ]; then
+         [ -f "$IGconf_connect_authkey" ] || { echo "Invalid keyfile" >&2; exit 1; }
+      fi
+  customize-hooks:
+    - |-
+      set -eu
+      if [ -n "${IGconf_connect_authkey-}" ] ; then
+         uchroot "$1" 'mkdir -m 0700 -p ${HOME}/.config/com.raspberrypi.connect/'
+         uchroot "$1" 'umask 077; cat > "${HOME}/.config/com.raspberrypi.connect/auth.key"' < "$IGconf_connect_authkey"
+      fi
+    - uchroot "$1" 'mkdir -p ${HOME}/.config/systemd/user/default.target.wants'
+    - uchroot "$1" 'ln -sf /usr/lib/systemd/user/rpi-connect-signin.service ${HOME}/.config/systemd/user/default.target.wants/rpi-connect-signin.service'
```

Comment:
As above re being in `.config/systemd/user/rpi-connect.service.wants` rather than `.config/systemd/user/default.target.wants`.